lib/turn.rb

```diff
@@ -0,0 +1,82 @@
+class Turn
+  attr_reader :player1,
+              :player2,
+              :spoils_of_war
+
+   def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = []
+   end
+
+  def type
+      #how the game is played. must have a winner or war! if two war than spoils of war! rank_of_cards_at
+      #get discarded with no winner.
+    if @player1.deck.rank_of_card_at(0) == @player2.deck.rank_of_card_at(0) && @player1.deck.rank_of_card_at(2) == @player2.deck.rank_of_card_at(2)
+      :mutually_assured_destruction
+    elsif @player1.deck.rank_of_card_at(0) == @player2.deck.rank_of_card_at(0)
+      :war
+    elsif @player1.deck.rank_of_card_at(0) != @player2.deck.rank_of_card_at(0)
+        :basic
+
+    end
```

Comment:
The order of checks in this conditional branching makes a lot of sense to me-- if both conditions aren't met for mutually assured destruction, check for the war condition. if that isn't met, it must be a basic turn. Nice! 🎉